ZAPD/GameConfig.h

```diff
@@ -21,6 +21,18 @@ class ExternalFile
 	ExternalFile(fs::path nXmlPath, fs::path nOutPath);
 };
 
+// Stores data from the XML file, the integer is the index (via ATOI) and the string is the value
+class CutsceneData
```

Comment:
This should be named something that doesn't make someone think its the actual data.

Reply:
I renamed that to ``CutsceneEnumData``, let me know if you think of a better name